picard/ui/options/dialog.py

```diff
@@ -84,6 +84,7 @@ class OptionsDialog(PicardDialog, SingletonDialog):
     autorestore = False
 
     options = [
+        config.TextOption("persist", "options_last_active_page", []),
```

Comment:
Since that's a text option, default value should be empty string, not empty list.

Reply:
Thanks, fixed